Products/CMFPlone/static/tinymce-AtD-plugin/support/unignore.html

```diff
@@ -5,8 +5,8 @@
 
 <p>Select a phrase to unignore it:</p>
 
-<SCRIPT TYPE="text/javascript" SRC="atdphrases.js"></SCRIPT>
```

Comment:
I think this script is no longer used and must be removed form the code on a different pull request; let's ask the @plone/framework-team.